### PR TITLE
Install codejail from pypi

### PR DESCRIPTION
**Pull request by aht007**
Issue: [BOM-2481](https://openedx.atlassian.net/browse/BOM-2481)

Description:
As codejail is released on PyPI, we have to install it from there which was previously being installed using github

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: requirements/base.in [hunk 1]
```diff
@@ -7,5 +7,4 @@ newrelic
 path.py
 requests
 six
-
--e git+https://github.com/edx/codejail.git@4127fc4bd5775cc72aee8d7f0a70e31405e22439#egg=codejail
+edx-codejail
```
Comment (iamsobanjaved) [inline requirements/base.in:10]: Just move this to the second position in the list to preserve alphabetic order

**Review by iamsobanjaved — APPROVED** [review]
Just an order suggestion otherwise 👍

File: requirements/base.in (hunk 1 above)
Comment (aht007) [reply requirements/base.in]: Done

**Comment by timmc-edx** [comment]
Reverted, see CR-4317 (internal ticket).

**Pull request merged**